travis-ci/fedora-build.reqs

```diff
@@ -0,0 +1,44 @@
+ gcc
```

Comment:
I think it'd be better to install the build dependencies via `dnf build-dep systemd` and leave this file for non-standard dependencies like `libasan`.

Reply:
Good idea, will fix.